uftrace.c

```diff
@@ -153,6 +154,7 @@ static struct argp_option uftrace_options[] = {
 	{ "kernel-only", OPT_kernel_only, 0, 0, "Dump kernel data only" },
 	{ "flame-graph", OPT_flame_graph, 0, 0, "Dump recorded data in FlameGraph format" },
 	{ "sample-time", OPT_sample_time, "TIME", 0, "Show flame graph with this sampling time" },
+	{ "graphviz", OPT_graphviz, 0, 0, "Dump recorded data in DOT format" },
```

Comment:
You also need to update the documentation.

Reply:
IMHO, `--dot` looks simpler than `--graphviz`.